documentation/source/user_section/command-line/deepseg/seg_ms_sc_mp2rage.rst

```diff
@@ -3,9 +3,5 @@
 seg_ms_sc_mp2rage
 =================
                 
-.. argparse::
-   :ref: spinalcordtoolbox.scripts.sct_deepseg.get_parser
-   :prog: sct_deepseg -h
-   :markdownhelp:
-   :noepilog:
-                
\ No newline at end of file
+This model has been retired. Please switch to the contrast-agnostic spinal cord model for further segmentation of MP2RAGE spinal cords.
```

Comment:
Maybe it should be a link? But at least let's format it so that people know we're talking about a model called `spinalcord`.
```suggestion
This model has been retired. Please switch to the contrast-agnostic ``spinalcord`` model for further segmentation of MP2RAGE spinal cords.
```

Reply:
Yes, very good point! Now that we have individual pages for each model (that are actually useful), we can link to specific tasks in much the same way we link to specific CLI arguements).